test/avocado/run-tests

```diff
@@ -149,6 +149,7 @@ def copy_avocado_logs(machine, title):
             # compress the logs and attach that
             archive = os.path.join(arg_attachments_dir, "{0}.tar.gz".format(dir))
```

Comment:
This `format(dir)` will behave differently now that `dir` is a full path.
I will implement a fix.